README.md

```diff
@@ -1,13 +1,22 @@
 # Meetups App
 
-## First run
-1. Create `.env` file in the root dir with fields from `.env.example`
-2. Optionally (to use maps) [get and set](https://yandex.ru/dev/jsapi-v2-1/doc/ru/#get-api-key) yandex maps api key or, even better, [google maps api key](https://developers.google.com/maps/documentation/javascript/get-api-key) in the `.env` file
-3. Install docker-compose
-4. `chmod +x entrypoint.sh && ./entrypoint.sh` (You can also use entrypoint-mac.sh if you are using mac, but be sure to install psycopg lib before that)
+## Basic setup
+1. Create `.env` file in the root dir with filled out fields from `.env.example`
+2. Install docker-compose for setting up local postgres db and application itself
+3. Run `docker-compose -f deployments/docker-compose.yml up -d`
+4. Go to `http://localhost:8000/swagger` and you should see the app running
 
-This will install postgres db in docker, install geo dependencies, create superuser admin:admin, run migrations and run server.
-5. After you have done this once you can just run `python manage.py runserver` next time
+## First run for local development (will install dependencies locally)
+1. Create `.env` file in the root dir with filled out fields from `.env.example`
+2. Install docker-compose for setting up local postgres db
+3. Run your postgres db via `docker-compose -f deployments/docker-compose-db.yml up -d`
+4. For linux: install geo dependencies via `sudo apt-get install binutils libproj-dev gdal-bin`
+5. For mac: install geo dependencies via `brew install binutils proj gdal`
+6. Setup your virtual environment and install requirements via `pip install -r requirements/local.txt`
+7. Install pre-commit hooks via `pre-commit install` for code formatting and linting before commit
```

Comment:
Looks like "prep-steps" or "project setup"

Reply:
Yes, however it seems like these procedures go together, and there is really no reason to separate them from the local setup at this point